src/app/guards/OnboardGuard.tsx

```diff
@@ -34,14 +36,26 @@ function OnboardGuard({ children, forceDisplay, grantsMutate }: Props) {
         return null;
     } else if (forceDisplay || status !== 'fulfilled') {
         return (
-            <FullPageWrapper fullWidth={true}>
-                <LocalZustandProvider createStore={localStore}>
-                    <BetaOnboard
-                        directive={directive}
-                        status={status}
-                        mutate={grantsMutate}
-                    />
-                </LocalZustandProvider>
+            <FullPageWrapper fullWidth>
+                <Grid
+                    container
+                    sx={{
+                        p: 2,
+                    }}
+                >
```

Comment:
Eventually we're going to have a grid on all the login pages so using it here to make sure it works.